packages/components/src/components/Icon/icons.ts

```diff
@@ -91,9 +91,9 @@ export const ICONS = {
     SHOW: require('../../images/icons/show.svg'),
     SHUFFLE: require('../../images/icons/shuffle.svg'),
     SIGNATURE: require('../../images/icons/signature.svg'),
-    T1: require('../../images/icons/t1.svg'),
-    TT: require('../../images/icons/tt.svg'),
-    TR: require('../../images/icons/tr.svg'),
+    '1': require('../../images/icons/1.svg'),
```

Comment:
I would love to call it `T1B1` and `T2T1`

I cannot call it `TT` and `T1` because `<Icon icon={deviceModel} />` it would not simply work with `t2b1` mix